lib/narn-lib.js

```diff
@@ -22,45 +39,69 @@ exports.detectYarn = function detectYarn(args) {
 
 exports.getYarnArgs = narnArgs => narnArgs;
 
-exports.getNpmArgs = narnArgs => {
-  const yarnArgs = minimist(narnArgs, { boolean: ["dev", "D"] });
+exports.getNpmArgs = (narnArgs = [], isPnpm = false) => {
```

Comment:
I added the isPnpm boolean helper since [pnpm has update-interactive implemented into it](https://pnpm.js.org/en/cli/update#interactive-i)